frontend/src/app/features/work-packages/components/wp-buttons/wp-create-button/wp-create-button.component.ts

```diff
@@ -79,6 +78,10 @@ export class WorkPackageCreateButtonComponent extends UntilDestroyedMixin implem
     // Find the first permission that is allowed
     this.authorisationService
       .observeUntil(componentDestroyed(this))
+      .pipe(
+        filter((links) => !!links.work_package),
```

Comment:
The fix seems to work correctly but it breaks some permission related specs. Adding the following change, will keep the fix and also not break the other specs.

```suggestion
        filter((links) => !!links.work_package || !links.work_package),
```